OpenRA.Game/Support/AssemblyLoader.cs

```diff
@@ -0,0 +1,394 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+#if !MONO
```

Comment:
This IMO feels a bit too hidden and easy to overlook here. How about moving this inside the namespace, with a comment explaining that the AssemblyLoader class is not used/usable under mono?

Reply:
Does this even build under mono?

Reply:
I doubt it, thats what i meant by useable. We should still be able to compile a file with usings and an "empty" namespace though right? Or will the style checker get mad?

Reply:
also note in the csproj file:
```
  <ItemGroup Condition="'$(Mono)' == ''">
    <PackageReference Include="Microsoft.DotNet.PlatformAbstractions" Version="3.1.6"  />
    <PackageReference Include="Microsoft.Extensions.DependencyModel" Version="5.0.0-preview.3-runtime.20214.6" />
    <PackageReference Include="System.Runtime.Loader" Version="4.3.0" />
  </ItemGroup>
```

Reply:
Ah, I see. Ok, can we still get a comment before the `#if !MONO` though?